test/testfiles/bridge-unmanaged-dns.json

```diff
@@ -0,0 +1,24 @@
+{
+	"container_id": "6ce776ea58b5",
+	"container_name": "testcontainer",
+	"networks": {
+		"podman": {
+			"interface_name": "eth0",
+			"static_ips": ["10.88.0.2"]
+		}
+	},
+	"network_info": {
+		"podman": {
+			"dns_enabled": true,
+			"driver": "bridge",
+			"id": "53ce4390f2adb1681eb1a90ec8b48c49c015e0a8d336c197637e7f65e365fa9e",
+			"internal": false,
+			"ipv6_enabled": false,
+			"name": "podman",
+			"network_interface": "brtest0",
+			"options": {
+				"mode": "unmanaged"
+			}
```

Comment:
I am surprised this is even valid like this, I would expect we need to define a subnet here, otherwise we cannot assign a ip address to the container.